### PR TITLE
[PROCEDURES] Point organization document to issue handling for binding procedure

### PR DESCRIPTION
**Pull request by dannon**

### CODE REVIEW

**Comment by dannon** [comment]
@jmchilton This is what you had in mind re: pointer from procedures-locked organization doc, right?

File: doc/source/project/organization.rst [hunk 1]
```diff
@@ -175,8 +175,23 @@ The Galaxy *committers* group may only commit directly to Galaxy (i.e. outside
...
+Issues (bugs, feature requests, etc.) should be reported at ISSUE_REPORT_, and
+handling of issues follows the procedures described in ISSUES_.
+
+
 * Patches for serious security vulnerabilities.
 * Cherry-picking and/or merging of existing approved commits to other branches.
 
 .. _LICENSE: https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt
 .. _CODE_OF_CONDUCT: https://github.com/galaxyproject/galaxy/blob/dev/CODE_OF_CONDUCT.md
+.. _ISSUES: https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst
```
Comment (jmchilton) [inline doc/source/project/organization.rst]: probably mean https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst

File: doc/source/project/organization.rst [hunk 2]
```diff
@@ -175,8 +175,23 @@ The Galaxy *committers* group may only commit directly to Galaxy (i.e. outside
 of a pull request and not following the procedures described here) the
 following two categories of patches:
```
Comment (jmchilton) [inline doc/source/project/organization.rst:176]: This text is inserted in the middle of a thought (two categories and then the list).

Comment (dannon) [inline doc/source/project/organization.rst:176]: Oops.  This is what i get for screwing up the original commit and re-applying it quickly :)   Good catch.

File: doc/source/project/organization.rst (hunk 1 above)
Comment (dannon) [inline doc/source/project/organization.rst]: Not sure what you mean?  Is that not the link?

Comment (dannon) [inline doc/source/project/organization.rst]: Hah, now I see.  We both did the same thing -- not organization.rst but issues.rst

Comment (jmchilton) [inline doc/source/project/organization.rst]: So yesterday we even agreed on the same typos.

Comment (martenson) [inline doc/source/project/organization.rst]: this is getting seriously scary

Comment (dannon) [inline doc/source/project/organization.rst]: Tell me about it, heh.

**Comment by dannon** [comment]
Ready for folks to weigh in.  Has been unchanged in 12 days and can be merged w/ 25% contributor +1.

**Comment by hexylena** [comment]
Pretty harmless. :+1:

**Comment by jxtx** [comment]
Makes a strong case for a lazy consensus provision...

+1

**Comment by remimarenco** [comment]
"Users can vote for issues by commenting with a +1."

:+1: :smile:

**Comment by martenson** [comment]
:+1:

**Comment by jmchilton** [comment]
We can't merge this because it was modified after opened. I think this is why people were quiet. If you open a new one I bet it will get the required votes right away.

**Comment by dannon** [comment]
Reopening, even if it is a bit silly, since I can't change the procedure without opening another procedures PR anyway.

**Comment by dannon** [comment]
All, please see #1020 for voting/merge, but I guess we have to wait a week anyway.

**Comment by hexylena** [comment]
@jmchilton we should really change those rules:
- changes should be allowed until people vote. 
  - We often have healthy discussions on procedures PRs. 
  - If people are withholding votes until discussion is done, I see no problem with that.
- once votes are made changes aren't allowed (or if they are made those votes are invalid (or) the PR becomes invalid)

I think we're just adding noise for people subscribed to this repo and ending up with 2x the number of procedures PRs we really need.

**Comment by dannon** [comment]
@erasche Yep, we just came to a similar conclusion in #galaxyproject.  I'm opening a second procedures PR in a sec with a few proposed amendments.

**Comment by hexylena** [comment]
That's what I get for not watching IRC :)

**Comment by jmchilton** [comment]
None of these rules represent what I wanted. I wanted 4 votes and a couple days for a procedural change. Don't make me defend any of this.

**Comment by jxtx** [comment]
We can always change it. 

(link to old discussion PR since it took me a bit to find: https://github.com/galaxyproject/galaxy/pull/295)

**Pull request closed without merging**